package com.eolwral.osmonitor.core;

message osInfo {
  
  required uint64 uptime = 1;           /**< system uptime */
  required uint64 totalMemory = 2;      /**< total memory */
  required uint64 freeMemory = 3;       /**< free memory */
  required uint64 sharedMemory = 4;     /**< shared memory */
  required uint64 bufferedMemory = 5;   /**< buffered memory */
  required uint64 cachedMemory = 6;     /**< shared memory */
  required uint64 totalSwap = 7;        /**< total swap size */
  required uint64 freeSwap = 8;         /**< free swap size */
}